### DISCUSSION TITLE
help to fix error

### DISCUSSION THREAD

**Discussion by venusky** [prisma/prisma#22336] [Q&A]
### Question

how can I solve this error? 
![image](https://github.com/prisma/prisma/assets/36695969/bb16ea3f-ef47-4978-8464-7f5b937b7885)
am working with nextjs and my model is this: 
```prisma
model Partenaire{
    id Int @id @default(autoincrement())
    createdAt DateTime @default(now())
    nom String
    slug String @unique
    programs Programme[] //relation vers le model programme
    import Import?
}
```
### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @venusky 👋 

Thank you for raising this question.

The error message you're seeing suggests that there is a type error in your code. It seems like you're trying to access the property `partenaire` on the prisma instance which does not exist according to the PrismaClient's type definition. This could mean that there is a typo in your code, or that the `partenaire` model is not defined in your Prisma schema. Did you run `npx prisma generate` to generate the Prisma client?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
